phpunit.xml.dist

```diff
@@ -2,7 +2,7 @@
 
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/7.2/phpunit.xsd" colors="true"
-    bootstrap="tests/bootstrap.php">
+    bootstrap="tests/bootstrap.php" executionOrder="random" resolveDependencies="true">
```

Comment:
The execution order should never matter unless there is unwanted interaction between tests (so it can be random)

Reply:
This was found by trying out [Infection](https://infection.github.io/). (I wasn't aware PHPUnit 7.2 had added this as an option.)

Reply:
Seems very promising. Always been interesting in the idea but never got around to trying it out. It finds some false positives, but does find some cases where there's no coverage.